.github/workflows/test.yml

```diff
@@ -269,7 +273,7 @@ jobs:
     steps:
     - uses: actions/checkout@v4
       with:
-        ref: ${{ github.event.pull_request.head.ref }}
+        ref: ${{ github.event.pull_request.head.sha }}
     - uses: actions/download-artifact@v4
```

Comment:
I think this needs to be changed back to `ref`. It worked fine before the `sha`.

Reply:
I'll try reverting. Seems weird since it did not work before